events.user.js

```diff
@@ -16,9 +16,12 @@ const stripesGradient = (colors, width, angle) => {
   let pos = 0;
 
   colors.forEach(color => {
+    let color2 = color ;
```

Comment:
Not loving `color2` as a variable name. Maybe we could use `original_color` and plain old `color`? Or `color` and `final_color`?